front/src/applications/stdcmV2/hooks/usePathfindingIsPossible.ts

```diff
@@ -0,0 +1,99 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { compact } from 'lodash';
+import { useSelector } from 'react-redux';
+
+import { osrdEditoastApi, type PathfindingResult } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+
+import useInfraStatus from 'modules/pathfinding/hooks/useInfraStatus';
+import { getSupportedElectrification, isThermal } from 'modules/rollingStock/helpers/electric';
+import usePathProperties from 'modules/pathfinding/hooks/usePathProperties';
+
+const usePathfindingIsPossible = () => {
```

Comment:
Since this usecase for a `uic === -1` is specific to stdcm, maybe we can rename this hook `useStdcmPathfinding` ? This way, we know we should use it elsewhere.

Reply:
Is it ok for you if we keep it like this ? Right now, this hook is only used in stdcm, but it could also be used in another part of the application

Reply:
Ok fine but I still don't like the name very much :) We could do `useSimplifiedPathfinding` or `useStaticPathfinding` (meaning it doesn't update the store contrary to the other)

Reply:
I renamed it useStaticPathfinding :white_check_mark: